src/vcpkg/base/messages.cpp

```diff
@@ -490,6 +490,20 @@ namespace
     StdErrMessageSink stderr_sink_instance;
 }
 
+namespace vcpkg
+{
+
```

Comment:
```suggestion
```

Also it seems like these should be moved into the order that they are next to the other MessageSink members.